README.md

```diff
@@ -93,6 +93,31 @@ scrypto test
 
 **Note:** The commands use the default account as transaction sender.
 
+## Playing with the simulator
+We provide a docker image so that you can build your blueprints 
+
+### Using local docker image
+The Dockerfile in the root of the repo should be work to build a docker image which will contain all the dependencies needed to be able build a blueprint using scrypto. 
+
+Build the docker image like. From the repo root
+```
+docker build -t radixdlt/simulator .
+```
+
+Then build your package by just running
+```
+docker run -v <your-blueprint-directory>:/src radixdlt/simulator
```

Comment:
`<path-to-your-scrypto-crate>`

Reply:
and `sudo docker`?

Reply:
the use of sudo to run docker really depends on each installation. docker actually suggest to not run it with sudo :)